modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/CompareCheckboxTag.java

```diff
@@ -40,6 +40,12 @@ public int doStartTag() throws JspException {
 				(CommerceContext)httpServletRequest.getAttribute(
 					CommerceWebKeys.COMMERCE_CONTEXT);
 
+			if ((commerceContext != null) &&
+				(commerceContext.getCommerceChannelId() == 0)) {
+
+				return SKIP_BODY;
```

Comment:
This check will fail on `commerceContext == null` and the runtime will continue with no `commerceContext`.

Is this what we want? If not, a simple
```
if (commerceContext == null || commerceContext.getCommerceChannelId() == 0) { ... }
```
will do the job.